Libraries/oneTBB/tbb-resumable-tasks-sycl/sample.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Add a sample GUID**
Go to [GUID Generator](https://www.guidgenerator.com/)

1. Checkbox 'Uppercase' and 'hyphens' field
2. Copy the value that is like : 825BEB2D-C9FA-4527-8770-9F7D765AAAAA

In your sample.json, enter a field called 
"guid" at the very top and make it : 
```c++
>  "guid" : "825BEB2D-C9FA-4527-8770-9F7D765AAAAA"
```